src/it/resources/org/checkstyle/suppressionxpathfilter/interfacetypeparametername/InputXpathInterfaceTypeParameterNameOnlySingleLetter.java

```diff
@@ -0,0 +1,7 @@
+package org.checkstyle.suppressionxpathfilter.interfacetypeparametername;
+
+public class InputXpathInterfaceTypeParameterNameOnlySingleLetter {
+  interface FirstInterface<T> {} // ok
+  interface SecondInterface<t> {} // ok
+  interface ThirdInterface<Type> {} // warn
```

Comment:
Please remove ok comments, it is just noise.
We need only warn to be instantly visible, as red flag.

Please remove from all inputs of your PR